CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -635,7 +635,7 @@ msgstr ""
 
 #. 3.2->settings-schema.json->menu-label->description
 #. 2.6->settings-schema.json->menu-label->description
-msgid "'Menu'-Button label"
+msgid "'Menu'-Button label (shown next to menu icon)"
```

Comment:
This will be in a Tab called "Panel", which should make it Mord clear, what is meant